expfmt/expfmt.go

```diff
@@ -26,7 +26,7 @@ const (
 
 	// The Content-Type values for the different wire protocols.
 	FmtUnknown      Format = `<unknown>`
-	FmtText         Format = `text/plain; version=` + TextVersion
+	FmtText         Format = `text/plain; charset=utf-8; version=` + TextVersion
```

Comment:
We put it at the end usually: https://github.com/prometheus/client_python/blob/master/prometheus_client/exposition.py#L28